data/variants.txt

```diff
@@ -627,7 +628,6 @@ ship "Cruiser" "Cruiser (Heavy)"
 		"Fragmentation Grenades" 20
 		"Armageddon Core"
 		"Dwarf Core"
-		"Supercapacitor"
```

Comment:
Can't wait to loot these Cruisers without any batteries and therefore no way to fire their particle cannons. :)
Might've been better to remove a radar jammer, downgrade the ion steering, or add another outfits expansion.

Reply:
LOL oops! How did I miss that this was its only battery!?